Builds/CMake/RippledRelease.cmake

```diff
@@ -2,12 +2,6 @@
    package/container targets - (optional)
 #]===================================================================]
 
-# Early return if the `containers` directory is missing,
-# e.g. when we are building a Conan package.
-if(NOT EXISTS containers)
-  return()
-endif()
-
```

Comment:
If this worked for you, then it is safe to remove. I think I added it when I was debugging Conan packaging in a personal branch. We're not there yet, though.